js/privatebin.js

```diff
@@ -1230,18 +1248,7 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             $errorMessage = $('#errormessage');
             $loadingIndicator = $('#loadingindicator');
             $statusMessage = $('#status');
-
-            // display status returned by php code, if any (e.g. paste was properly deleted)
-            var serverStatus = $statusMessage.text();
```

Comment:
In my tests the opposite was the case: Due to this all errors or status messages of the server resulted in exceptions. The template ensures these are not hidden and work even if there is an error in the JS.